roles/openshift_logging/defaults/main.yml

```diff
@@ -26,6 +26,9 @@ openshift_logging_kibana_proxy_debug: false
 openshift_logging_kibana_proxy_cpu_limit: null
 openshift_logging_kibana_proxy_memory_limit: null
 openshift_logging_kibana_replica_count: 1
+openshift_logging_kibana_cert: ""
+openshift_logging_kibana_key: ""
+openshift_logging_kibana_ca: ""
```

Comment:
Can you add a comment explaining that these should be local file paths on the control host?